base/centos-8/install.sh

```diff
@@ -23,8 +23,15 @@ export LANG=en_US.utf8
 yum -y update && yum -y install wget sudo epel-release make
 yum -y install ansible python3-requests python3-jmespath
 
+# map os arch to busybox arch labels as needed
```

Comment:
upstream removes this busybox work in https://github.com/splunk/docker-splunk/pull/509/files

Reply:
removed this per upstream